packages/frontend/web-ui/src/game/models/GameOptions.ts

```diff
@@ -0,0 +1,9 @@
+export interface GameOptions {
```

Comment:
Let's remove this in favor of existing API models

Reply:
Removed!